lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -894,7 +901,8 @@ void MtohRenderOverride::_RemovePanel(MString panelName)
     }
 
     if (_renderPanelCallbacks.empty()) {
-        ClearHydraResources();
+        static const bool fullReset = false;
```

Comment:
With modern C++ this should be
```suggestion
        constexpr bool fullReset = false;
```